engine/install/rhel.md

```diff
@@ -0,0 +1,271 @@
+
+- Most users
+  [set up Docker's repositories](#install-using-the-repository) and install
+  from them, for ease of installation and upgrade tasks. This is the
+  recommended approach.
+
+- Some users download the RPM package and
+  [install it manually](#install-from-a-package) and manage
+  upgrades completely manually. This is useful in situations such as installing
+  Docker on air-gapped systems with no access to the internet.
+
+- In testing and development environments, some users choose to use automated
+  [convenience scripts](#install-using-the-convenience-script) to install Docker.
+
+### Install using the repository
+
+Before you install Docker Engine for the first time on a new host machine, you need
+to set up the Docker repository. Afterward, you can install and update Docker
+from the repository.
+
+#### Set up the repository
+
+{% assign download-url-base = "https://download.docker.com/linux/rhel" %}
+
+Install the `yum-utils` package (which provides the `yum-config-manager`
+utility) and set up the **stable** repository.
+
+```console
+$ sudo yum install -y yum-utils
+
+$ sudo yum-config-manager \
+    --add-repo \
+    {{ download-url-base }}/docker-ce.repo
+```
+
+> **Optional**: Enable the **nightly** or **test** repositories.
+>
+> These repositories are included in the `docker.repo` file above but are disabled
+> by default. You can enable them alongside the stable repository.  The following
+> command enables the **nightly** repository.
+>
+> ```console
+> $ sudo yum-config-manager --enable docker-ce-nightly
+> ```
+>
+> To enable the **test** channel, run the following command:
+>
+> ```console
+> $ sudo yum-config-manager --enable docker-ce-test
+> ```
+>
+> You can disable the **nightly** or **test** repository by running the
+> `yum-config-manager` command with the `--disable` flag. To re-enable it, use
+> the `--enable` flag. The following command disables the **nightly** repository.
+>
+> ```console
+> $ sudo yum-config-manager --disable docker-ce-nightly
+> ```
+>
+> [Learn about **nightly** and **test** channels](index.md).
+
+#### Install Docker Engine
+
+1.  Install the _latest version_ of Docker Engine and containerd, or go to the next step to install a specific version:
+
+    ```console
+    $ sudo yum install docker-ce docker-ce-cli containerd.io
+    ```
+
+    If prompted to accept the GPG key, verify that the fingerprint matches
+    `060A 61C5 1B55 8A7F 742B 77AA C52F EB6B 621E 9F35`, and if so, accept it.
+
+    > Got multiple Docker repositories?
+    >
+    > If you have multiple Docker repositories enabled, installing
+    > or updating without specifying a version in the `yum install` or
+    > `yum update` command always installs the highest possible version,
+    > which may not be appropriate for your stability needs.
+
+    Docker is installed but not started. The `docker` group is created, but no users are added to the group.
```

Comment:
```suggestion
This command installs Docker, but it doesn't start Docker. It also creates a Docker group, however, it doesn't add any users to the group by default. 
```

Reply:
The majority of this page is the same for all  the RPM installs; I kept it for now, but we can change it for all of them if you want (perhaps as a follow-up).